rest-server/src/components/user/__tests__/user_joiValidation.test.ts

```diff
@@ -25,24 +25,24 @@ beforeAll(async () => {
   await syncProjectTables();
   token = await request(app)
     .post(signupUrl)
-    .send({ email: 'newuser@gmail.com', password: 'howdy17' })
+    .send({ email: 'ok@gmail.com', password: 'howdy17' })
     .then(response => token = response.header.authorization)
 })
 
 describe('GET/api/users', () => {
   test('should return 400 if parameter is not valid email', async () => {
     const response = await request(app)
```

Comment:
are you using response here ?

Reply:
on line 29? Yeah to get the token

Reply:
I meant on line 34, you store the promise in the response const but never use it anywhere ..... it’s nit picky but was wondering